app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+
+    source_file, destination_path = parts[1], parts[2]
```

Comment:
Use unpacking and avoid indexing to retrieve values for better code style. You should check that command starts with `mv` as well